lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -147,7 +173,7 @@ def my_tracing_endpoint(self) -> Optional[str]:
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
 
-LIBPATCH = 4
+LIBPATCH = 5
 
 PYDEPS = ["opentelemetry-exporter-otlp-proto-http>=1.21.0"]
```

Comment:
If we import scenario, we also start being dependent on scenario here, right?

Reply:
good point. Will get rid of that.